tools/webpack/vendors.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const { join } = require( 'path' );
+
+/**
+ * WordPress dependencies
+ */
+const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+
+const copiedVendors = {
+	'inert-polyfill': [
```

Comment:
I wonder if we can import inert-polyfill as well to avoid a specific way to build this vendor.

Reply:
I just realized that this polyfill is not used anymore in Gutenberg, so I removed the build and removed it from package.json